extensions/amp-date-countdown/validator-amp-date-countdown.protoascii

```diff
@@ -30,31 +30,35 @@ tags: {  # <amp-date-countdown>
   tag_name: "AMP-DATE-COUNTDOWN"
   requires_extension: "amp-date-countdown"
   attrs: {
-    name: "end-date"
-    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+    name: "biggest-unit"
+    value_regex_casei: "days|hours|minutes|seconds"
   }
   attrs: {
-    name: "timestamp-ms"
+    name: "end-date"
     mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+    value_regex: "\\d{4}-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d(:[0-5]\\d(\\.\\d+)?)?(Z|[+-][0-1][0-9]:[0-5][0-9])"
   }
   attrs: {
-    name: "timestamp-seconds"
-    mandatory_oneof: "['end-date', 'timestamp-ms', 'timestamp-seconds']"
+    name: "locale"
+    value_regex_casei: "de|en|es|fr|id|it|ja|ko|nl|pt|ru|th|tr|vi|zh-cn|zh-tw"
   }
   attrs: {
     name: "offset-seconds"
+    value_regex: "\\d{10}"
```

Comment:
Maybe something like "-?\\d{,10}", to include smaller integers and negative numbers.